queries/facility/highlights.scm

```diff
@@ -0,0 +1,82 @@
+[
+  ";"
+  "."
+  ","
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+]  @punctuation.bracket
+
+[
+  (comment)
+  (doc_comment)
+] @comment
+
+("method") @keyword.function
+
+[
+  "service"
+  "errors"
+  "data"
+  "enum"
+  "extern"
+] @type.builtin
+
+(type) @type.builtin
```

Comment:
Won't `(type name: (identifier))` override this?

```suggestion
```

Reply:
This query still applying to primitives https://facilityapi.github.io/define/fsd#fields

`(type name: (identifier))` is selecting user defined types (dtos, enums, etc).

![Screenshot 2023-11-27 at 10 44 42 AM](https://github.com/nvim-treesitter/nvim-treesitter/assets/9691017/4ec32485-4b8b-44cf-8bbf-34dd725d6ca0)

In this example, `Widget` is a dto defined in the service definition.